gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -199,15 +214,15 @@ def run(
         relative_pose_priors: Dict[Tuple[int, int], PosePrior] = {},
         scale_factor: float = 1.0,
         gt_wTi_list: Optional[List[Optional[Pose3]]] = None,
-    ) -> Tuple[List[Optional[Point3]], Optional[GtsfmMetricsGroup]]:
+    ) -> Tuple[List[Optional[Pose3]], Optional[GtsfmMetricsGroup]]:
         """Run the translation averaging.
 
         Args:
             num_images: number of camera poses.
             i2Ui1_dict: relative unit-translation as dictionary (i1, i2): i2Ui1
             wRi_list: global rotations for each camera pose in the world coordinates.
             absolute_pose_priors: priors on the camera poses (not delayed).
-            relative_pose_priors: priors on the pose between camera pairs (not delayed)
+            relative_pose_priors: priors on the pose between camera pairs (not delayed) as (i1, i2): i1Ti2.
```

Comment:
Rename to `i1Ti2_priors`

Reply:
Now renamed to `i2Ti1_priors`, as it looks like we are pulling out `i2Ti1` from the dictionary.